### PR TITLE
unrar: add page

### PR DESCRIPTION
**Pull request by joelhy**
Add unrar command to common page.

### CODE REVIEW

**Comment by joelhy** [comment]
origin from https://github.com/tldr-pages/tldr/pull/486

**Comment by felixonmars** [comment]
:+1:

File: pages/common/unrar.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# unrar
+
+> Extract RAR archives
+
+- extract files with full path
```
Comment (waldyrious) [inline pages/common/unrar.md]: I'm not sure what "with full path" means here. Can you clarify?

File: pages/common/unrar.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`unrar x {{compressed.rar}}`
+
+- test archive files
```
Comment (waldyrious) [inline pages/common/unrar.md]: The description should be "test an archive file", to match the specific example below.

Comment (joelhy) [inline pages/common/unrar.md]: Comments clarified.

File: pages/common/unrar.md (hunk 1 above)
Comment (joelhy) [inline pages/common/unrar.md]: Add an  `unrar e` command to pair with `unrar x` and clarify usages.

File: pages/common/unrar.md (hunk 2 above)
Comment (joelhy) [inline pages/common/unrar.md]: Comments carified.

**Comment by rubenvereecken** [comment]
:+1:, I like the use of the `.rar` postfix and the first comment explaining how directory structure is lost, that one always gets me. Good being explicit there. 

Thanks!

**Pull request merged**